packages/mask/src/plugins/Collectible/SNSAdaptor/NFTPage.tsx

```diff
@@ -1,22 +1,19 @@
 import type { SocialAddress, NetworkPluginID, SocialIdentity } from '@masknet/web3-shared-base'
 import { CollectionList } from '../../../extension/options-page/DashboardComponents/CollectibleList'
-import { useCurrentVisitingProfile } from '../hooks/useContext'
 
 export interface NFTPageProps {
     identity?: SocialIdentity
     socialAddress?: SocialAddress<NetworkPluginID>
 }
 
 export function NFTPage({ socialAddress, identity }: NFTPageProps) {
-    const currentVisitingProfile = useCurrentVisitingProfile()
-
     if (!socialAddress) return null
 
     return (
         <CollectionList
             addressName={socialAddress}
             persona={identity?.publicKey}
-            visitingProfile={currentVisitingProfile}
+            twitterId={identity?.identifier?.userId.toLowerCase()}
```

Comment:
I think the original edition is better. All NextID applications should start from a `SocialIdentity`.